### PR TITLE
[FIX] Cannot edit or delete custom sounds

### PR DESCRIPTION
**Pull request by ccfang**
@RocketChat/core 

Just a minor bug when i would like to upload custom sound and find no way to delete it.

### CODE REVIEW

File: packages/rocketchat-custom-sounds/client/admin/soundInfo.js [hunk 1]
```diff
@@ -90,15 +90,15 @@ Template.soundInfo.onCreated(function() {
 
 	this.autorun(() => {
 		const data = Template.currentData();
-		if (data.clear != null) {
+		if (data != null && data.clear != null) {
```
Comment (karlprieb) [inline packages/rocketchat-custom-sounds/client/admin/soundInfo.js]: I think you can change that to `if (data && data.clear !== null)` :)

File: packages/rocketchat-custom-sounds/client/admin/soundInfo.js [hunk 2]
```diff
@@ -90,15 +90,15 @@ Template.soundInfo.onCreated(function() {
...
 			this.clear = data.clear;
 		}
 	});
 
 	this.autorun(() => {
 		const data = Template.currentData();
 		const sound = this.sound.get();
-		if (sound.name != null) {
+		if (sound != null && sound.name != null) {
```
Comment (karlprieb) [inline packages/rocketchat-custom-sounds/client/admin/soundInfo.js]: Same here

File: packages/rocketchat-custom-sounds/client/admin/soundInfo.js (hunk 1 above)
Comment (ccfang) [reply packages/rocketchat-custom-sounds/client/admin/soundInfo.js]: Sure, thank you for your comment.

File: packages/rocketchat-custom-sounds/client/admin/soundInfo.js (hunk 2 above)
Comment (ccfang) [reply packages/rocketchat-custom-sounds/client/admin/soundInfo.js]: OK.
By the way, i just let it the same with the `emojiInfo.js`

**Comment by graywolf336** [comment]
@karlprieb do you mind reviewing again?

**Pull request merged**